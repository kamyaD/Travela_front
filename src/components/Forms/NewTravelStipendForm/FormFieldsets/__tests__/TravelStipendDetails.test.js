import React from 'react';
import TravelStipendFieldset from '../TravelStipendDetails';

const props = {
  values: {
    centers: [{
      location: 'Nairobi, Kenya'},
    {
      location: 'Kigali, Rwanda'
    }],
    stipend: 1234
  },
  modalType: 'create travel stipend',
};

const setup = (props) => shallow(<TravelStipendFieldset {...props} />);

describe('<TravelStipendFieldset />', () => {
  it('should render properly', () => {
    const wrapper = setup(props);
    expect(wrapper.length).toBe(1);
  });

  it('should match snapshot', () => {
    const wrapper = setup(props);
    expect(wrapper).toMatchSnapshot();
  });
});
