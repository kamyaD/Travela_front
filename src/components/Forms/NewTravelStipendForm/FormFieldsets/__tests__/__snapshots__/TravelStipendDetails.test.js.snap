// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TravelStipendFieldset /> should match snapshot 1`] = `
<fieldset
  className="personal-details"
>
  <div>
    <div>
      <div
        className="input-group"
      >
        <div
          className="spaces"
        >
          <Input
            autoComplete="off"
            choices={Array []}
            className="request{ uniq }dropdown stipend-location"
            id="user-location"
            label="Select Location"
            labelNote=""
            name="center"
            onChange={null}
            size=""
            type="dropdown-select"
          />
        </div>
        <div
          className="spaces"
        >
          <Input
            autoComplete="off"
            className="request{ uniq }dropdown stipend-amount "
            id="your-manager"
            label="Enter Amount in Dollars ($)"
            labelNote=""
            max="1000"
            min="1"
            name="stipend"
            onChange={[Function]}
            onInvalid={[Function]}
            placeholder="1000"
            size=""
            type="number"
          />
          <span
            className="hide-error"
          >
            Amount should be a positive integer and not more than 1000
          </span>
        </div>
      </div>
    </div>
  </div>
</fieldset>
`;
